script/testing/oltpbench/run_oltpbench.py

```diff
@@ -62,17 +62,29 @@ def generate_test_suite(args):
             # there is no loop
             oltp_test_suite.append(TestCaseOLTPBench(oltp_testcase_base))
 
-    return oltp_test_suite
+    # read server commandline args in config files
+    server_args_json = oltp_test_suite_json.get("server_args")
+    if server_args_json:
+        server_args = ""
+        for attribute,value in server_args_json.items():
+            server_args = server_args + " " + "-" + str(attribute) + "=" + str(value)
```

Comment:
Just for readability it might help to change to `'{SERVER_ARGS} -{ATTRIBUTE}={VALUE}'.format(SERVER_ARGS=server_args,ATTRIBUTE=attribute,VALUE=value)`